src/coreclr/src/jit/valuenum.h

```diff
@@ -228,6 +228,9 @@ class ValueNumStore
     template <typename T, typename NumMap>
     inline ValueNum VnForConst(T cnsVal, NumMap* numMap, var_types varType);
 
+    // returns true if vn is known to be a contant number > 0
```

Comment:
constant is mispelled, better would be:

`// returns true if vn is known to be a constant int32 that is > 0`